src/app/node-data/template.html

```diff
@@ -150,6 +150,8 @@
 
         <div class="os-options">
           <mat-checkbox [id]="Controls.DynamicConfig"
+                        [matTooltip]="isDynamicKubletConfigSupported()? '' : 'Kubernetes version 1.24 and higher do not support dynamic kubelet configuration.'"
```

Comment:
Better to have global constant for Kubernetes version and use that variable instead of adding inline` 1.24` as mentioned here:

https://github.com/kubermatic/dashboard/pull/4514/files#r891301253